compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -177,6 +199,9 @@ impl<'local, 'interner> Interpreter<'local, 'interner> {
         } else if let Some(oracle) = func_attrs.oracle() {
             if oracle == "print" {
                 self.print_oracle(arguments)
+            // Ignore debugger functions
+            } else if oracle.starts_with("__debug") {
+                Ok(Value::Unit)
```

Comment:
Would it be difficult to support these? It seems that we should error if they're completely unsupported at comptime

Reply:
Not supporting these prevents the comptime tests from running successfully in `execution_success` since we also copy those for the debugger tests which insert all of these functions.
My thinking for passing through without doing anything for these is that the debugger is for debugging at runtime while this is a separate compile-time step.